modules/assim.sequential/R/met_filtering_helpers.R

```diff
@@ -15,8 +25,18 @@ sample_met <- function(settings,nens=1){
   tmp_members <- gsub(paste0(".", end_date),   "", tmp_members) 
   member_names <- unique(dirname(ens_members))
   
-  #this will change from model to model, generalize later
-  ens_ind <- unlist(sapply(paste0(member_names, ".clim"), grep, tmp_members))
+  # this will change from model to model, generalize later
+  # This function is temporary but if we will continue to use this approach for met ensembles (instead of met process workflow)
```

Comment:
This whole function seems to be a bit of a hack -- any time we write a function that as a hardcoded if on model type we've done something wrong. The next PR to SDA needs to get rid of (or correctly generalize) this function all together, and NOT replace it with sample_met.model, because if done correctly there's should be nothing about the process of sampling from a list of inputs that should require model-specific code. Likewise, the sampling needs to be generalized to be able to sample over all inputs, not just met, as I really hope we'll have soil ensembles in place within the year.